open-craft/opencraft
# SE-2870 - Changed Heartbeat App Addition

The heartbeat app addition was done using a condition that checked if the `openedx_release` is *not* `master`. However, releases after `ironwood`, such as `juniper` will have the heartbeat app added by default. 

Therefore, the condition was changed to check for old releases instead of checking if *not* `master`. However, not all old releases are included, only the following:
* `ironwood`
* `hawthorn`
* `ginkgo`

**JIRA tickets**: SE-2870

**Merge deadline**: None

**Testing instructions**:

1. Clone `open-craft/opencraft` and checkout the branch `nizar/change_heartbeat_app_addition`
2. Following the [OCIM docs](https://ocim.opencraft.com/en/latest/installation/) to install OCIM on your machine.
3. Run OCIM and create a new instance running `open-release/juniper.1`
4. Launch a new app server and make sure it configures successfully.

**Author notes and concerns**:

There is a problem with the `integration_tests_group_1` as mentioned in SE-2872. So it will probably fail. However, we can place this into `External / Blocker` after review until SE-2872 is resolved.

**Reviewers**
